agntcy_acp/langgraph/acp_node.py

```diff
@@ -192,27 +199,114 @@ def _handle_run_output(self, state: Any, run_output: RunOutput):
             run_error: RunError = run_output.actual_instance
             raise ACPRunException(f"Run Failed: {run_error}")
         elif isinstance(run_output.actual_instance, RunInterrupt):
-            raise ACPRunException(f"ACP Server returned a unsupporteed interrupt response: {run_output}")
+            # This case is handled in the invokes
+            pass
+
         else:
-            raise ACPRunException(f"ACP Server returned a unsupporteed response: {run_output}")
+            raise ACPRunException(
+                f"ACP Server returned a unsupporteed response: {run_output}"
+            )
 
         return state
 
-    def invoke(self, state: Any, config: RunnableConfig) -> Any:
+    def _get_thread_id(self, config):
```

Comment:
Should we add types to the args here and in the `_handle_interrupt` declarations?